msg/Material.msg

```diff
@@ -4,3 +4,4 @@ float32[3] color  # RGB for visualization node
 float32 min_return  # 0.0 to 1.0 representing the minimum chance of return
 float32 max_return  # 0.0 to 1.0 representing the maximum chance of return
 float32 angular_return  # the chance of return lost per degree from perpendicular
+float32 intensity # set the scan intensity of material
```

Comment:
should this type be `float64` since it's used as a `double` elsewhere in the code?

Reply:
yeah I will make it float64. Thanks.